go.sh

```diff
@@ -19,10 +20,10 @@ for CMD in flockerctl flocker-ca flocker-deploy flocker-config flocker-install f
     cat <<EOF |sudo tee /usr/local/bin/${PREFIX}${CMD} >/dev/null
 #!/bin/sh
 DEPRECATED="${DEPRECATED}"
```

Comment:
The `go.sh` script prints the deprecation warning when it installs the unnofficial flocker tools.
It also adds the deprecation warning to each of the deprecated wrapper scripts that it generates.
If it didn't reassign that variable inside the wrapper script, the generated script would end up having weird multiline echo statements, like:
```
if [ "\${DEPRECATED}" = "TRUE" ]; then
    echo "WARNING: ${PREFIX}${CMD} was deprecated in Flocker 1.14.0 and will be removed in future versions of Flocker.
Use the official installation methods and tools instead.
See https://docs.clusterhq.com." >&2
    echo "" >&2
fi
```